docker/build.sh

```diff
@@ -63,6 +63,7 @@ CmakeGenerate() {
     -S "$SOURCES_DIR" \
     -B "$BUILD_DIR" \
     -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
+    -DCMAKE_UNITY_BUILD="ON" \
```

Comment:
little nitpick, the `""` aren't needed in this script
```suggestion
    -DCMAKE_UNITY_BUILD=ON \
```